File: node/calc.js
function add(a, b) {
  return a + b;   // ✅
} // addition

function sub(a, b) {
  return a - b;   // ✅ (уже правильно)
} // subtraction

function mul(a, b) {
  return a * b;   // ✅
} // multiplication

function div(a, b) {
  if (b === 0) throw new Error('division by zero'); // ✅
  return a / b;
}

module.exports = { add, sub, mul, div };

File: ruby/calc.rb
def add(a, b)
  a + b
end  # addition

def sub(a, b)
  a - b
end  # subtraction

def mul(a, b)
  a * b
end  # multiplication

def div(a, b)
  raise ZeroDivisionError, "division by zero" if b == 0
  a.to_f / b
end